src/App.js

```diff
@@ -1,23 +1,27 @@
-import logo from './logo.svg';
 import './App.css';
+import Auth from './Views/Auth/Auth';
+import { BrowserRouter, Redirect, Route, Switch } from 'react-router-dom';
+import { useState } from 'react';
+import { getUser } from './services/auth';
+import Todos from './Views/Todos/Todos';
+import Header from './Views/Comp/Header';
 
 function App() {
+  const [currentUser, setCurrentUser] = useState(getUser());
+
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      <BrowserRouter>
+        <Header setCurrentUser={setCurrentUser} currentUser={currentUser}/>
+        <Switch>
+          <Route exact path = "/">
+            {currentUser ? <Todos /> : <Redirect to = "/auth"/>}
+          </Route>
+          <Route exact path = "/auth">
+            <Auth setCurrentUser={setCurrentUser}/>
+          </Route>
```

Comment:
Here we will want to check for a current user so that authenticated users do not have access to sign-in.